### ISSUE TITLE
macOS Sequoia showes below everytime when I open

### ISSUE THREAD

**Issue by Lelouch-Vi-Britannia-CC** [k-pet-group/BlueJ-Greenfoot#2398]
2024-09-16 16:46:01.694 java[2023:63977] +[IMKClient subclass]: chose IMKClient_Legacy
2024-09-16 16:46:01.694 java[2023:63977] +[IMKInputSession subclass]: chose IMKInputSession_Legacy

**Comment by neilccbrown** [comment]
It looks like this might be a macOS oddity as many different applications (not all of them Java) are reporting seeing it:

 - https://github.com/dotnet/maui/issues/24794
 - https://github.com/airsdk/Adobe-Runtime-Support/discussions/3475
 - https://discussions.apple.com/thread/255761734?sortBy=rank

Hopefully Apple will fix it in an update.

**Comment by alsonick** [comment]
Yeah, I'm also seeing this everytime I open BlueJ, hopefully Apple pushes a fix soon.

<img width="1094" alt="Screenshot 2024-10-10 at 14 02 42" src="https://github.com/user-attachments/assets/aec895cd-e690-4713-8289-23becad6cc0b">

**Comment by neilccbrown** [comment]
There is also now a JDK bug report where they also seem to conclude it's an Apple bug: https://bugs.openjdk.org/browse/JDK-8340330

**Comment by roefer** [comment]
It would be nice if BlueJ had some kind of filter for the error output based on a regular expression. This way, it would be easy to suppress such an annoying output by a simple entry in the `bluej.properties`.

**Comment by mkolling** [comment]
Similar problem here. For me the text is:

2025-01-30 13:07:50.652 java[33779:458960] +[IMKClient subclass]: chose IMKClient_Modern
2025-01-30 13:07:50.652 java[33779:458960] +[IMKInputSession subclass]: chose IMKInputSession_Modern
